ci/ray_ci/docker_container.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from ci.ray_ci.container import Container, _DOCKER_ECR_REPO
+from ci.ray_ci.builder_container import PYTHON_VERSIONS
+from ci.ray_ci.utils import docker_pull
+
+
+class DockerContainer(Container):
+    def __init__(self, python_version: str) -> None:
+        super().__init__(
+            "forge",
+            volumes=[
+                f"{os.environ['RAYCI_CHECKOUT_DIR']}:/rayci",
+                "/var/run/docker.sock:/var/run/docker.sock",
+            ],
+        )
+        self.python_version = python_version
+
+    def run(self) -> None:
+        base_image = (
+            f"{_DOCKER_ECR_REPO}:{os.environ['RAYCI_BUILD_ID']}-raypy38cu118base"
+        )
+        docker_pull(base_image)
+        wheel_name = (
+            "ray-3.0.0.dev0-"
+            f"{PYTHON_VERSIONS[self.python_version]['bin_path']}-"
```

Comment:
get a var for `PYTHON_VERSIONS[self.python_version]['bin_path']` and try to get the string to a single line?

also in other places.

I think in general, I find it easier if the thing embedded in the f string is closer to a single var reference.